include/rapid_pbd/editor.h

```diff
@@ -8,6 +8,7 @@
 #include "rapid_pbd_msgs/EditorEvent.h"
 #include "rapid_pbd_msgs/Program.h"
 #include "rapid_pbd_msgs/Step.h"
+#include "rapid_pbd_msgs/ProgramNotifier.h"
```

Comment:
Be sure to use clang-format to format your code. For example, it will alphabetize the #includes.